document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoCollection.java

```diff
@@ -417,6 +448,11 @@ private BasicDBObject selectionCriteriaForKey(Key key) {
     return new BasicDBObject(ID_KEY, key.toString());
   }
 
+  private BasicDBObject selectionCriteriaForKeys(Set<Key> keys) {
```

Comment:
Can't this be private? A simple inspection tells me yes it does but I am not 100% sure. Tho `ID_KEY` is static.

Reply:
I think you meant to ask about static right? Fixing it.

Reply:
yeah I meant static, sorry.

Reply:
Raised https://github.com/hypertrace/document-store/pull/25